openwisp_radius/api/views.py

```diff
@@ -162,25 +194,27 @@ def authenticate_user(self, request, user):
         a valid user password or a valid user token
         can be overridden to implement more complex checks
         """
-        return user.check_password(
-            request.data.get('password')
-        ) or self.check_user_token(
-            request, user
-        )  # noqa
+        return bool(
+            user.check_password(request.data.get('password'))
+            or self.check_user_token(request, user)
+        )
```

Comment:
Also, user can bypass the `can_auth` flag status and authenticate with password.
But I think that's correct behavior because if they have the password, the can get a new token anyway!